cli/azd/pkg/tools/dotnet/dotnet.go

```diff
@@ -40,6 +40,19 @@ type dotNetCli struct {
 	commandRunner exec.CommandRunner
 }
 
+type ResponseContainerConfiguration struct {
+	Config ResponseContainerConfigurationExpPorts `json:"config"`
+}
+
+type ResponseContainerConfigurationExpPorts struct {
```

Comment:
if this is only used within the tools package, do we want to export them?  or just keep them lower case and within the tools package?